addons/product/models/product_template.py

```diff
@@ -390,15 +390,8 @@ def _set_packaging_ids(self):
             if len(p.product_variant_ids) == 1:
                 p.product_variant_ids.packaging_ids = p.packaging_ids
 
-    @api.depends('type')
```

Comment:
keep the dependency, as it's needed on most overrides, and doesn't cost anything here.